CMakeLists.txt

```diff
@@ -167,69 +181,79 @@ if(POTENTIAL_FLAGS)
 	endif()
 endif()
 
-###
 # Generate the config header
-###
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake_in" "${CMAKE_CURRENT_BINARY_DIR}/config.h")
-include_directories("${CMAKE_CURRENT_BINARY_DIR}")
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake_in"
+			   "${CMAKE_CURRENT_BINARY_DIR}/config.h")
 
-###
 # Build the library
-###
 add_subdirectory(src)
 
-###
 # Make the pkg-config and config files
-###
+#
 # needed for openal.pc.in and openal-config.in
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix "\${prefix}")
-set(libdir ${CMAKE_INSTALL_FULL_LIBDIR})
-set(bindir ${CMAKE_INSTALL_FULL_BINDIR})
-set(includedir ${CMAKE_INSTALL_FULL_INCLUDEDIR})
+set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
```

Comment:
This regresses a fix in #11.

If one uses an absolute path for this, e.g. `-DCMAKE_INSTALL_LIBDIR=/lib64`, this will be put in the .pc file.

```
libdir=${exec_prefix}//lib65
```

I know it's not DRY or autotools-like but most of the CMake-based projects I encounter regularly install .pc files with full paths in it:

```
prefix=/usr
libdir=/usr/lib64
```

Reply:
oh, good point, i'll undo that part of my change.